theseus/utils/examples/pose_graph/dataset.py

```diff
@@ -47,7 +49,7 @@ def read_3D_g2o_file(path: str) -> Tuple[int, List[th.SE3], List[PoseGraphEdge]]
 
                 x_y_z_quat = torch.from_numpy(
                     np.array([tokens[3:10]], dtype=np.float64)
-                )
+                ).to(dtype)
```

Comment:
Why not change the line above to do `, dtype=dtype)`?

Reply:
It looks like [`torch.from_numpy`](https://pytorch.org/docs/stable/generated/torch.from_numpy.html) has no `dtype` options.

Reply:
But there is a `dtype` inside the `np.array` which we can use. I think torch will respect the type of the underlying numpy array.

Reply:
I know. But the `dtype` in `np.array()` does not accept types like `torch.float32`.

Reply:
Why can't you do something like this

![image](https://user-images.githubusercontent.com/4759586/161059113-137d7164-e37e-429f-9ef1-b746e14ac37e.png)

Reply:
Yes, I can do this. But `dtype` is one the input of [`read_3D_g2o_file`](https://github.com/facebookresearch/theseus/blob/bed363f5c1643ffa146c57d0ddde80dd5e28fc5b/theseus/utils/examples/pose_graph/dataset.py#L32). Is it OK to mix the use of `torch.dtype` and `np.dtype`?